addons/mail/tests/test_res_users_settings.py

```diff
@@ -18,6 +38,7 @@ def test_find_or_create_for_user_should_create_record_if_not_existing(self):
 
     @users('employee')
     def test_find_or_create_for_user_should_return_correct_res_users_settings(self):
+        self.user_employee.res_users_settings_ids.unlink()
```

Comment:
It's a bit confusing, maybe just create a portal user here instead?

Reply:
Sorry but I don't want to lose time to adapt tests for this kind of reason. It should work no matter what, since a portal user could be transformed into an internal user, or some user may not have settings before migration. The test is legit, you can change it latter if you want.